portal/views/portal.py

```diff
@@ -948,11 +953,14 @@ def contact():
         abort(400, "No contact request body provided")
     body = u"From: {sendername}<br />Email: {sender}<br /><br />{body}".format(
         sendername=sendername, sender=sender, body=formbody)
-    recipients = current_app.config['CONTACT_SENDTO_EMAIL']
+    recipient = request.form.get('type')
```

Comment:
Hmm - this worries me.

Perhaps this sounds overly paranoid... but I know contact forms are frequently used to generate spam - seems like we're now opening up an email gateway, if we blindly take the recipient as a form variable.  I'd suggest either validating it's a known email address, or passing something used to look up the correct contact.

Reply:
Ah, that's a good point. I'll validate that the email actually belongs to an org.

Reply:
Modified the logic; now it only uses the recipient from the form if it matches an existing `Organization.email`; otherwise, it just always uses the default from the config